llvm/lib/Target/RISCV/RISCVRegisterInfo.td

```diff
@@ -28,21 +28,21 @@ class RISCVReg16<bits<5> Enc, string n, list<string> alt = []> : Register<n> {
   let AltNames = alt;
 }
 
-def sub_16 : SubRegIndex<16>;
```

Comment:
There's nothing FPR-y about the index itself though? It just happens to only be applicable for FPRs.